plugins/ccgx/ccgx.quirk

```diff
@@ -57,6 +57,7 @@ Plugin = ccgx
 GType = FuCcgxDmcDevice
 Summary = Dock Management Controller Device
 ParentGuid = USB\VID_03F0&PID_0363
+Vendor = HP
```

Comment:
How will this appear in gnome-firmware list of devices? Does Vendor string get prepended? What I mean to confirm is whether it will appear as "HP USB-C Dock G5", which is good, or HP HP USB-C Dock G5" which is not?

Reply:
"HP USB-C Dock G5" is displayed.

I attached pictures when the Thunderbolt Dock G4 is connected to gnome-firmware.
You can two "HP Thunderbolt Dock G4"s , one is G4 dock and the other is TBT controller in G4 Dock. 

1. G4 Dock (DMC). It is a part of infineon plugin.
![HP G4 Dock Name](https://user-images.githubusercontent.com/32348354/165895567-56807a04-6626-44b0-96a3-6844949006b2.png)

2. TBT controller in the G4 Dock (DMC). It is not a part of infineon plugin.
![HP G4 TBT Name](https://user-images.githubusercontent.com/32348354/165895752-b80267de-ec9c-47fb-85a3-41f830bb6f52.png)

Reply:
![G5 dock name](https://user-images.githubusercontent.com/32348354/165897958-85f4d606-c6e6-4882-9636-3bc69a100163.png)